tools/cldr-code/src/main/java/org/unicode/cldr/test/ExampleGenerator.java

```diff
@@ -1362,8 +1375,33 @@ private void handleFormatUnit(XPathParts parts, String unitPattern, List<String>
         if (parts.getElement(-1).equals("unitPattern")) {
             String count = parts.getAttributeValue(-1, "count");
             DecimalQuantity amount = getBest(Count.valueOf(count));
+
             if (amount != null) {
-                addFormattedUnits(examples, parts, unitPattern, shortUnitId, amount);
+                if (shortUnitId.equals("light-speed")) {
+                    Map<String, String> overrideMap =
+                            ImmutableMap.of(parts.toString(), unitPattern);
+                    // add examples showing usage
+                    examples.add("Used as a fallback in the following:");
+                    CLDRFileOverride cldrFileOverride = new CLDRFileOverride(cldrFile, overrideMap);
+                    addFormattedUnitsConstructed(
+                            cldrFileOverride, examples, parts, "light-speed-second", amount);
+                    addFormattedUnitsConstructed(
+                            cldrFileOverride, examples, parts, "light-speed-minute", amount);
+                    addFormattedUnitsConstructed(
+                            cldrFileOverride, examples, parts, "light-speed-hour", amount);
+                    addFormattedUnitsConstructed(
+                            cldrFileOverride, examples, parts, "light-speed-day", amount);
+                    addFormattedUnitsConstructed(
+                            cldrFileOverride, examples, parts, "light-speed-week", amount);
+                    addFormattedUnitsConstructed(
+                            cldrFileOverride, examples, parts, "light-speed-month", amount);
+                    examples.add("Compare with:");
```

Comment:
Will this "Compare with:" get displayed appropriately on the front end?

Reply:
yes